### DISCUSSION TITLE
Code Folding with 6 Lines of Vanilla JavaScript | /en/2023/09/code-folding/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1514] [Comments]
Do you know what the top requested feature of blogdown has been in the past six years? Code folding. I have rarely seen 34 upvotes on a Github issue in projects that I maintain.
[&hellip;] The &hellip;

https://yihui.org/en/2023/09/code-folding/

**Comment by chuxinyuan** [comment]
我想给代码块上增加一个复制按钮，当鼠标放在代码块上时才出现，这个功能在代码块比较长的情况下比较方便，类似[Manjaro 论坛](https://forum.manjaro.org/t/pamac-only-showing-flatpaks/147018/5)的这种效果。

**Reply by yihui** [reply]
走你：https://yihui.org/en/2023/09/copy-button/

6 行 JS 代码加 10 行 CSS 就足以搞定了。

**Comment by albertjin** [comment]
To correct a trivial typo :)

> For example, if you *want fold* tables

=> `want to fold`

**Reply by yihui** [reply]
Thanks! Fixed via 1a79b92f0541b1c518c51f8e56d6ff13e4ce72ce. Next time please feel free to use the menu "Edit this page" at the bottom of the post to send a Github pull request to fix any typo you find.
